bot.py

```diff
@@ -295,6 +295,16 @@ def cmd_uptime(match: Match[str]) -> Response:
     return UptimeResponse()
 
 
+@handle_message('!pep')
```

Comment:
and maybe handle 4 digit thing here with regex

Reply:
I'm just doing what Anthony said in chat. And I don't know regex.